vivisect/__init__.py

```diff
@@ -2758,9 +2760,20 @@ def loadFromFile(self, filename, fmtname=None, baseaddr=None):
             return self.normFileName(filename)
 
         mod = viv_parsers.getParserModule(fmtname)
+
+        # get baseaddr and size, then make sure we have a good baseaddr
+        baseaddr, size = mod.getMemBaseAndSize(self, filename=filename, baseaddr=baseaddr)
+        logger.debug('initial baseva: 0x%x  size: 0x%x', baseaddr, size)
+        if baseaddr == 0:
+            baseaddr = 0x300000
+
+        baseaddr = self.findFreeMemoryBlock(size, baseaddr)
```

Comment:
we may wish to add an address-spacer arg, like alignment, which would allow us to make different bins end up in a different multiple of 0x10000000 or something.